### PR TITLE
S3 to efs copy + automount efs mount point to lambda

### PR DESCRIPTION
**Pull request by SimonBeckx**
- S3 to efs copy in one step
- Automount EFS mount point to lambda.
- Moved EFS creation before the Lambda stack creation
- ECS mount path

### CODE REVIEW

File: templates/EFS.yml [hunk 1]
```diff
@@ -42,6 +42,24 @@ Resources:
...
+        Uid: "1000"
+        Gid: "1000"
+      RootDirectory:
+        Path: {{ filesystem.s3_to_efs_copy.efs_directory_path }}
+        CreationInfo:
+          OwnerGid: "1000"
+          OwnerUid: "1000"
+          Permissions: "0755"
+{%   endif %}
+
```
Comment (rik2803) [inline templates/EFS.yml:62]: Suggestion:
* Change `1000` to `0`
* Remove the `RootDirectory` part

File: templates/EFS.yml [hunk 2]
```diff
@@ -70,6 +88,162 @@ Resources:
...
+      Role: !GetAtt LambdaEfsRole{{ filesystem.cfn_name }}.Arn
+      Code:
+        S3Bucket: "{{ lambda_function_bucket_name }}"
+        S3Key: "{{ filesystem.s3_to_efs_copy.lambda_key }}"
+      Runtime: python3.8
+      Environment:
+        Variables:
+          PROJECT: "{{ application }}"
+          ENVIRONMENT: "{{ env }}"
+          FILESYSTEM: "{{ filesystem.cfn_name }}"
```
Comment (rik2803) [inline templates/EFS.yml:161]: Does the function really need/use this envvar?

Comment (SimonBeckx) [reply templates/EFS.yml:161]: These variables are used to send the slack notification

**Comment by SimonBeckx** [comment]
- lambda updated to make use of the `aws-lambda-s3-trigger-to-efs` lambda
- Tested and verified!

**Review by rik2803 — APPROVED** [review]

**Pull request merged**